mailpoet/assets/css/src/components-plugin/_landingpage.scss

```diff
@@ -30,6 +30,90 @@
     .mailpoet-faq-accordion {
       margin: 25px 0;
     }
+
+    .landing-faq-mobile {
+      display: none;
+    }
+
+    @include respond-to(small-screen) {
+      .landing-faq-header {
+        display: none;
+      }
+
+      .landing-faq-mobile {
+        display: block;
+      }
+    }
+  }
+
+  .landing-content {
+    .hero-section {
+      $hero-image-offset: 6rem;
+      background-color: $color-landingpage-background-light;
+      margin-top: $hero-image-offset;
+      padding: $content-padding;
+
+      .hero-image {
+        margin-top: -($hero-image-offset + 2rem);
+      }
+    }
+
+    .landingpage-images {
+      @include respond-to(medium-screen) {
+        width: 100%;
+      }
+    }
+
+    .landingpage-general-features {
+      p:last-child {
+        margin: 10px auto;
+        width: 60%;
+      }
+
+      .landingpage-feature-icon {
+        display: block;
+        margin: 0 auto;
+        padding: 20px;
+        text-align: center;
+      }
+
+      @include respond-to(medium-screen) {
+        p:last-child {
+          width: 100%;
+        }
+      }
+    }
+
+    .landingpage-wooCommerce-features {
+      margin-top: 30px;
+      padding: 2rem 10rem;
+
+      .landingpage-wooCommerce-feature-item {
+        padding: 30px;
+
+        @media screen and (min-width: 960px) and (max-width: 1390px) {
+          .landingpage-images {
+            width: 100%;
```

Comment:
Small thing, but this is how it displays on my M1 13in, the text overlaps a bit with the image. 
<img width="1176" alt="Screen Shot 2023-01-13 at 4 38 23 PM" src="https://user-images.githubusercontent.com/8002881/212432573-b6ca0d74-7f45-4eb3-b18b-edc313751e9e.png">
A bit smaller or bigger screen and there is no overlap.